pkg/dao/repositories.go

```diff
@@ -98,6 +99,28 @@ func (p repositoryDaoImpl) Update(repoIn RepositoryUpdate) error {
 	return nil
 }
 
+func (r repositoryDaoImpl) OrphanCleanup() error {
+	// lookup orphans
+	query := r.db.Model(&models.Repository{}).
+		Joins("left join repository_configurations on repositories.uuid = repository_configurations.repository_uuid").
+		Where("repository_configurations.uuid is NULL").
+		Where("repositories.public is false").
+		Where("repositories.created_at < (LOCALTIMESTAMP - INTERVAL '1 week' ) ").
+		Select("repositories.uuid")
+
+	// Delete orphans
+	tx := r.db.
+		Unscoped().
```

Comment:
Thats a good question, i had adapted the rpm orphan cleanup method and it has this.  I assume it was due to the soft delete: https://gorm.io/docs/delete.html#Delete-permanently  
but since we don't have a deleted_at field, i'm not sure why we need it.  I'll delete from both